extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/grpc/GrpcTracingServerInterceptor.java

```diff
@@ -66,17 +67,40 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(
 
     private static class GrpcServerNetServerAttributesGetter extends InetSocketAddressNetServerAttributesGetter<GrpcRequest> {
         @Override
-        public InetSocketAddress getAddress(final GrpcRequest grpcRequest) {
+        public String transport(final GrpcRequest grpcRequest) {
+            return SemanticAttributes.NetTransportValues.IP_TCP;
+        }
+
+        @Nullable
```

Comment:
I think we should remove `@Nullable`. We don't use it.

Reply:
done